spec/requests/sessions_spec.rb

```diff
@@ -0,0 +1,52 @@
+require "rails_helper"
+
+RSpec.describe "Sessions", type: :request do
```

Comment:
So this is interesting. Aren't you usually anchoring request specs around a specific controller?

```suggestion
RSpec.describe SessionsController, type: :request do
```

Reply:
In theory it's testing more than just the controller, and could be testing something across multiple controllers (although maybe not in these cases). 
The RSpec examples do something similar https://rspec.info/features/8-0/rspec-rails/request-specs/request-spec/

Reply:
Yeah, I actually had a look. I think I'm used the the idiom of 100% correlation in the backend apis